internal/xds/translator/listener.go

```diff
@@ -467,6 +504,27 @@ func (t *Translator) addHCMToXDSListener(xdsListener *listenerv3.Listener, irLis
 	return nil
 }
 
+func routeConfigName(irListener *ir.HTTPListener, useProtocolPortAsListenerName bool) string {
+	// If the ir Listener is HTTP, then the route is merged on the same port.
+	if useProtocolPortAsListenerName && irListener.TLS == nil {
+		return fmt.Sprintf("default-%d", irListener.ExternalPort)
```

Comment:
should this be `default-80` or `http-80` ? 
cc @envoyproxy/gateway-maintainers

Reply:
http-80 or combining both sounds good to me, default alone would be confusing

Reply:
`http-80` sounds better than `default-80`